linux.cc

```diff
@@ -386,9 +387,40 @@ static long sys_getcwd(char *buf, unsigned long size)
 // to Linux signature of this system call. The underlying ioctl function which we delegate to
 // is variadic and takes slightly different paremeters and therefore cannot be used directly
 // as other system call implementations can.
+#define __NCCS 19
```

Comment:
nitpick: maybe call this KERNEL_NCCS to make it clearer it's related to this kernel structure.
In general using double underscores in names is not a good idea - it suggest some builtin macro, which this isn't.

Reply:
Great idea!